mcserver/utils/misc.py

```diff
@@ -44,7 +45,8 @@
     'prevent-proxy-connections': False,
     'use-native-transport': True,
     'enable-rcon': False,
-    'motd': 'A Minecraft Server'
+    'motd': 'A Minecraft Server',
+    'favicon': None
```

Comment:
This constant should reflect the default minecraft [server.properties](https://server.properties)